saleor/graphql/product/types.py

```diff
@@ -35,17 +46,32 @@ def resolve_attribute_list(attributes):
     return attributes_list
 
 
+def resolve_attribute_value_type(attribute_value):
```

Comment:
Wouldn't that be more convenient to just define `type` on the `ProductAttributeValue`, instead of guessing what's the value?